server.js

```diff
@@ -4,23 +4,42 @@ const app = express();
 const bodyParser = require("body-parser");
 app.use(bodyParser.json());
 
-const mockUserData = [
-    { id: 1, name: "Mark" },
-    { id: 2, name: "Jill" },
+const mockUserData = [{
+        id: 1,
+        name: "Mark"
+    },
+    {
+        id: 2,
+        name: "Jill"
+    },
 ];
+//added message and status
+let function_message = '';
+let test_status = false;
```

Comment:
Camelcase would match the existing style of naming conventions

Reply:
Changed check pls!